File: est/toff/Task5.java
import java.util.Scanner;

public class Task5 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        long min = scan.nextLong();
        long max = scan.nextLong();
        scan.close();

        int count = 0;
        String numStr = min + "";
        int k = numStr.length();
        long num = min;
        int i = numStr.charAt(0) - '0';
        while (num <= max) {
            try {
                num = Long.parseLong(((i % 10) + "").repeat(k));
            } catch (Exception e) {
                break;
            }
            if (num > max) break;
            count++;
            i++;
            if (i % 10 == 0) {
                k++;
                i++;
            }
        }
        
        System.out.println(count);
    }

}

File: est/toff/Task5.py
min_val, max_val = input().split(" ")
min_val = int(min_val)
max_val = int(max_val)

count = 0
num_str = str(min_val)
k = len(num_str)
num = min_val
i = int(num_str[0])

while num <= max_val:
    try:
        num = int(str(i % 10) * k)
    except ValueError:
        break
    if num > max_val:
        break
    count += 1
    i += 1
    if i % 10 == 0:
        k += 1
        i += 1

print(count)
